src/pages/steps/AdditionalInformationStep/OrganizerPicker.tsx

```diff
@@ -40,6 +42,10 @@ const isUitpasOrganizer = (organizer: Organizer): boolean => {
   );
 };
 
+const isCultuurkuurOrganizer = (organizer: Organizer): boolean => {
```

Comment:
You don't need to define a return type here.
It's going to do it automatically.

```suggestion
const isCultuurkuurOrganizer = (organizer: Organizer) => {
```